types/react-loadable/webpack.d.ts

```diff
@@ -13,6 +13,7 @@ declare namespace LoadableExport {
     id: number;
     name: string;
     file: string;
+    publicPath: string;
```

Comment:
The relevant change: https://github.com/jamiebuilds/react-loadable/commit/60495e387c11e97a11a3087402afa589d822c0fc#diff-821a16d38cf4ff69edc6b5b48313aa0bR20

Reply:
fix test